app/models/payment/PaymentRecord.scala

```diff
@@ -40,49 +39,57 @@ case class PaymentRecord(reference: String,
 }
 
 object PaymentRecord {
+  val createdOnFormatter: DateTimeFormatter = DateTimeFormatter.ISO_DATE_TIME
+
+  val formatFullDtf: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
 
   private[payment] object DateFormatting {
     def formatFull(date: LocalDate)(implicit messages: Messages): String =
-      createDateFormatForPattern("d MMMM yyyy", messages).format(date.toDate)
+    createDateFormatForPattern("d MMMM yyyy", messages).format(date)
+
 
-    private def createDateFormatForPattern(pattern: String, messages: Messages): SimpleDateFormat = {
+    private def createDateFormatForPattern(pattern: String, messages: Messages): DateTimeFormatter = {
```

Comment:
As discussed this will need to be changed because Java does not seem to fully support a locale for Wales.